tests/api/cluster/test_access_to_s3buckets.py

```diff
@@ -0,0 +1,78 @@
+from model_mommy import mommy
+import pytest
+
+from controlpanel.api.cluster import (
+    App,
+    RoleGroup,
+    S3Bucket,
+    User,
+)
+
+@pytest.fixture(autouse=True)
+def enable_db_for_all_tests(db):
+    pass
+
+
+@pytest.fixture
+def bucket():
+    return mommy.make('api.S3Bucket')
```

Comment:
Do you need to use `model_mommy`? and also, do we need to persist the `S3Bucket` in the DB for this test? [`mommy.make()` will persist to DB](https://model-mommy.readthedocs.io/en/latest/basic_usage.html#non-persistent-objects) which is slow.

One test persisting wouldn't make much difference but in general we should avoid persisting to DB if not necessary as many of these tests cause the whole testsuite to be slow and increate the test feedback cycle.

Reply:
AFAICT these tests "just" check that the correct AWS calls are made, so probably you don't need to persist the various records to the DB - but I didn't try to tweak/run the tests.

Reply:
I benchmarked this and it makes no time difference to use `mommy.make` vs `mommy.prepare` vs just instantiating the model class and not persisting it. But I've switched to `mommy.prepare` to avoid persisting to the database anyway.